common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -134,6 +135,29 @@ public TachyonConf() {
       throw new RuntimeException("Unable to load default Tachyon properties file.", e);
     }
 
+    // Override default properties if there exist corresponding environment variables
+    if (getenv("TACHYON_MASTER_ADDRESS") != null) {
+      defaultProps.setProperty(Constants.MASTER_HOSTNAME, getenv("TACHYON_MASTER_ADDRESS"));
+    }
+    if (getenv("TACHYON_MASTER_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.MASTER_BIND_HOST, getenv("TACHYON_MASTER_BIND_HOST"));
+    }
+    if (getenv("TACHYON_MASTER_WEB_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.MASTER_WEB_BIND_HOST,
+          getenv("TACHYON_MASTER_WEB_BIND_HOST"));
+    }
+    if (getenv("TACHYON_WORKER_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.WORKER_BIND_HOST, getenv("TACHYON_WORKER_BIND_HOST"));
+    }
+    if (getenv("TACHYON_WORKER_DATA_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.WORKER_DATA_BIND_HOST,
+          getenv("TACHYON_WORKER_DATA_BIND_HOST"));
+    }
+    if (getenv("TACHYON_WORKER_WEB_BIND_HOST") != null) {
+      defaultProps.setProperty(Constants.WORKER_WEB_BIND_HOST,
+          getenv("TACHYON_WORKER_WEB_BIND_HOST"));
+    }
+
```

Comment:
Okay. It should be straightforward to switch into $TACHYON_JAVA_OPTS. The current binding strategy was trying to match Spark's design, which could simplify the further integration:
https://spark.apache.org/docs/latest/configuration.html#environment-variables